src/rustllvm/ArchiveWrapper.cpp

```diff
@@ -150,19 +158,40 @@ LLVMRustWriteArchive(char *Dst,
                      const LLVMRustArchiveMember **NewMembers,
                      bool WriteSymbtab,
                      Archive::Kind Kind) {
+
+#if LLVM_VERSION_MINOR >= 9
```

Comment:
Perhaps these should reference the old version, like the previous set? `LLVM_VERSION_MINOR <= 8` and flip the clauses?